assets/in

```diff
@@ -72,7 +73,15 @@ for branch in $fetch; do
   git branch $branch FETCH_HEAD
 done
 
+return_ref=""
+if [ -n "$tag_filter" ]; then
```

Comment:
Finally I use tag filter in the in command as well, as the tag (or `tag_filter`) might after check is called.